tests/coalaCITest.py

```diff
@@ -127,6 +127,20 @@ def test_fail_acquire_settings(self, debug=False):
             self.assertNotEqual(retval, 0,
                                 'coala was expected to return non-zero')
 
+    def test_additional_parameters_settings(self, debug=False):
+        with bear_test_module():
+            retval, stdout, stderr = execute_coala(
+                    coala.main, 'coala',
+                    '--non-interactive', '-S',
+                    'testSection.bears=SpaceConsistencyTestBear',
+                    'testSection.enabled=False',
+                    '-c', os.devnull,
+                    debug=debug)
+            self.assertFalse(stdout)
+            self.assertNotIn('During execution, we found that some', stderr)
+            self.assertNotEqual(retval, 0,
```

Comment:
shouldn't this be a clean run because the section passed is disabled?

Reply:
@Makman2 what do you mean by clean run?

Reply:
Meaning that coala returns 0 because no errors occurred.